scripts/docker/README.md

```diff
@@ -0,0 +1,190 @@
+# Borg Docker Image
+
+Docker image with [BorgBackup](https://borgbackup.readthedocs.io/en/stable/) client utility and sshfs support. Borg is a deduplicating backup program supporting compresion and encryption. It's very efficient and doesn't need regular full backups while still supporting data pruning.
+
+## Quick start
+
+First, pull the image to keep it up to date. Then create and run the borg backup container. In this quick start, the `/etc` and `/home` directories from the host are bind mounted to the container as read only. These are the directories which will be backed up. The backed up data will be stored in the `borg-repo` Docker volume, and the data will be protected with the `my-secret-pw` password. If the host is using SELinux, the `--security-opt label:disable` flag must be used, because we don't want to relabel the `/etc` and `/home` directories while we want the container to have access to them. After the backup is done, data will be pruned according to the default policy and checked for errors. Borg is running in a verbose mode within the container, so the detailed output from backup will be printed. At the end, the container is deleted. This is done by separate `docker rm` command, because the `--rm` option to the `docker run` would remove also the Docker volumes, and we don't want that. Deleting the container and pulling the image from registry every time keeps the container fresh every time the backup is run.
+
+```
+docker pull borgbackup/borg
+docker run \
+  -e BORG_REPO=/borg/repo \
+  -e BORG_PASSPHRASE=my-secret-pw \
+  -e BACKUP_DIRS=/borg/data \
+  -e EXCLUDE='*/.cache*;*.tmp;/borg/data/etc/shadow' \
+  -e COMPRESSION=lz4 \
+  -e PRUNE=1 \
+  -v borg-cache:/root/.cache/borg \
+  -v borg-repo:/borg/repo \
+  -v /etc:/borg/data/etc:ro \
+  -v /home:/borg/data/home:ro \
+  --security-opt label:disable \
+  --entrypoint=/usr/bin/borgbackup \
+  --name borg-backup \
+  borgbackup/borg
+docker rm borg-backup
```

Comment:
It probably makes sense to just run with `docker run --rm` so you don't have to run `docker rm` afterwards. This sample image doesn't save data in the container, so it's best to be a good citizen and remove it afterwards.

Reply:
it is using docker volumes, so it would delete the associated volume.